src/containers/AccountPanel.tsx

```diff
@@ -60,8 +60,10 @@ export class AccountPanelComponent extends React.Component<Props, State> {
 
   public handleStopEditing = () => {
     const { activeAccount } = this.props
+    const { name } = this.state
+    const trimmedName = name.trim()
 
-    const errorText = this.validateInput(this.state.name, ValidationType.unique)
+    const errorText = this.validateInput(name, ValidationType.unique)
```

Comment:
Should this use the trimmed name?

Reply:
nah, it needs to be the full name to check the length isn't over 20 chars inside the validation. We also check the trimmed version inside the validation anyway.